api/api-doc/storage_service.json

```diff
@@ -637,6 +637,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"sf",
+                     "description":"do not flush the memtables when taking the snapshot",
```

Comment:
I would phrase it like: when set to true, do not flush...
also sf is unclear (is that stand for skip flush? if so, you should probably use those wordings in the description)

But that's a personal taste, beside that, the rest looks good

Reply:
`sf` does stand for `skip flush`.  Like I wrote in the commitlog, I did try to call it `skip_flush` but couldn't make it to work with scylla-jmx.  Only when renaming it on both sides to `sf` the option passed correctly.

I'll clarify the description as per your suggestion.